apps/www/src/shapes/view.turtle

```diff
@@ -363,6 +343,8 @@ view:In rdfs:label "one of" .
     dash:editor dash:InstancesSelectEditor ;
     sh:minCount 1 ;
     sh:maxCount 1 ;
+    sh1:labelPath schema:name ;
+    sh1:setFocusNodeLabel true ;
```

Comment:
When `sh1:setFocusNodeLabel true`, the current focus node will be labeled instead. 

Here, it means the subject of this `view:cube` predicate.